server/server/models/user.py

```diff
@@ -86,11 +86,18 @@ def get_role(self):
         return self.roles.role
 
     @staticmethod
-    def update_user(request, json_data):
+    def update_user(request=None, json_data=None, data_success=True):
         """ Method to update user in database """
-        if request.dbsession.query(User).get(request.matchdict['profile_id']):
-            request.dbsession.query(User)\
-            .filter(User.id == request.matchdict['profile_id']).\
-            update(json_data)
+        response = {'success': False}
```

Comment:
It's a model, it has to know nothing about responses and format. It should only get validated data and put in DB and return True or False whether it was successful... nothing more. No HTTP codes, no dicts with success keys, onlt True or False. All logic should be implemented in views, not in models.